pages/content/amp-dev/documentation/courses/beginning-course/thinking-in-components@pt_BR.md

```diff
@@ -0,0 +1,265 @@
+---
+$title: Pensando em componentes
+$order: 4
+leveled: true
+---
+
+[filter level="beginner"]
+## Adicionando recursos ao nosso site
+
+Até agora, convertemos nosso site HTML básico em um site AMP básico. Há apenas um erro de validação restante em nosso site, e vamos corrigi-lo usando um componente `<amp-img>` em vez de uma tag `<img>`. Conforme corrigimos esse último erro de validação, aprenderemos o que são os componentes de AMP, por que algumas tags HTML foram substituídas ou não permitidas em AMP e como adicionar componentes ao nosso site.
```

Comment:
I leave this question to you and to @fellyph ... I assume that it's ok to use the Portuguese "componentes" throughout this article, even though we also use the English phrase "Web Components"?

Reply:
When the concept was about web components specifically I leave it in English, when the concept was generic as a "component" I translated it into Portuguese. Do you agree @fellyph?

Reply:
That makes sense to me! I've already delayed this enough, so I want to go ahead and approve. If Fellyph thinks this should be changed, well, that's still possible!